nri/index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8">
+    <title>NRIの強み　ビジネスモデルの特徴 | 企業情報 | 野村総合研究所(NRI)
+    </title>
```

Comment:
書いている内容は同じですが、ファイル内でできる限り記述は統一しましょう。
（現場だとこのあたりは指摘されないこともありますが、今回は指導的な観点もあり指摘しています。）

```suggestion
    <title>NRIの強み　ビジネスモデルの特徴 | 企業情報 | 野村総合研究所(NRI)</title>
```